openprescribing/dmd2/templates/dmd/vmp_relationships.html

```diff
@@ -35,14 +55,23 @@ <h2>Presentations</h2>
   {% endfor %}
 </table>
 {% else %}
-<p>There are no matching presentations.</p>
+<p>There are no matching BNF codes.</p>
 {% endif %}
 
-<h2>VMPPs, AMPs, AMPPs</h2>
-<table class="table table-condensed">
+<h2>Related VMPPs, AMPs, AMPPs</h2>
+<p>This table shows how the BNF codes above relate to each other, and to their DM+D equivalents.  Cells are coloured by BNF code; cells without colors are products in DM+D that don't have a corresponding BNF code (and have therefore never appeared in dispensing data).</p>
+<table class="table table-condensed" id="amp-table">
+  <tr>
+    <td></td>
+    <td></td>
+    <th colspan="{{ vmpp_cols }}" class="pp-header">
+      generic product packs
```

Comment:
Except it's not always generic!  I'd stick with "VMPPs".

Reply:
Isn't it? Aren't all VMPPs generic?